pkg/gitproviders/common.go

```diff
@@ -483,3 +473,12 @@ func NewUserRepositoryRef(domain, user, repoName string) gitprovider.UserReposit
 		},
 	}
 }
+
+func waitUntilRepoCreated(ownerType ProviderAccountType,owner, name string) error{
```

Comment:
Looks like a formatting miss here. I would advise configuring your editor to run `gofmt` on save. That ensures everyone's code looks roughly the same.